src/geometry/MergedMeshGroup.ts

```diff
@@ -231,6 +231,14 @@ export class MergedMeshGroup extends GeometryGroup {
   computeBoundingBox(matrix: THREE.Matrix4, box: THREE.Box3, treeIndex: number): THREE.Box3 {
     box.makeEmpty();
 
+    if (!this.treeIndexMap[treeIndex]) {
```

Comment:
Can this still happen after fix?

Reply:
Hopefully not. But I was thinking it is better that it gives a warning and does not crash, since this is a public function with no requirements or documentation that says it will throw if a bad treeIndex is used.